Keypad/Keypad.js

```diff
@@ -26,6 +27,10 @@ import Skinnable from '../Skinnable';
 
 import css from './Keypad.module.less';
 
+const Container = SpotlightContainerDecorator({
+	enterTo: 'default-element'
+}, 'div');
```

Comment:
I would avoid adding more layer for SpotlightContainer. 
You can create SpotlightContainerLayout by wrapping `Layout` so that we can reduce the layer